code/game/objects/items/storage/backpack.dm

```diff
@@ -352,33 +352,53 @@
 
 /obj/item/storage/backpack/duffelbag/cursed
 	name = "living duffel bag"
-	desc = "A cursed clown duffel bag that hungers for food of any kind. Putting some food for it to eat inside of it should distract it from eating you for a while. A warning label on one of the duffel bag's sides cautions against feeding your \"new pet\" anything poisonous..."
+	desc = "A cursed clown duffel bag that hungers for food of any kind."
 	icon_state = "duffel-curse"
 	inhand_icon_state = "duffel-curse"
-	slowdown = 1.3
+	slowdown = 2
+	item_flags = DROPDEL
 	max_integrity = 100
 	///counts time passed since it ate food
 	var/hunger = 0
 
-/obj/item/storage/backpack/duffelbag/cursed/Initialize()
+/obj/item/storage/backpack/duffelbag/cursed/examine(mob/user)
+	. = ..()
+	. += "<span class='danger'>A warning label suggests that it eats food inside.</span>"
+	. += "<span class='danger'>If that food happens to be a horribly ruined, burned mess the chef scrapped out of the microwave, then it might have negative effects on the bag...</span>"
```

Comment:
Shouldn't this text, being unconditional, just be part of the `desc`? Or are you trying for a specific message order here?

Reply:
It was in the description at one point, and bloated it out massively in a big wall of text. I'm definitely going for a specific message order here so people will recognize the important parts, and not have important information hidden in a block text box.

Reply:
You can use `\n` characters in the `desc` string though.